nova/volume/cinder.py

```diff
@@ -1006,8 +1006,15 @@ def migrate_by_connector(self, context, volume_id, connector):
                         'code': getattr(ex, 'code', None)})
             return False
 
-        # NOTE(jkulik): A day would be 24 MIB/s on average for a 2 TiB volume.
-        MIGRATION_TIMEOUT = 24 * 3600
+        assumed_speed_mib = CONF.cinder.min_migration_speed_mib_per_second
+        try:
+            volume = client.volumes.get(volume_id)
+            # volume.size is GiB and we assume 24 MiB/s for no good reason
```

Comment:
Isn't that comment now outdated?

Reply:
Yes, you're right. This also led to me changing the default timeout below, which was not 24h anymore. It now uses the `assumed_speed_mib` together with a 2 TiB volume to compute the timeout.